src/scatra_ele/4C_scatra_ele_calc_multiporo_reac.hpp

```diff
@@ -613,11 +613,11 @@ namespace Discret
       {
         const int numfluidphases = efluxnp.size();
 
-        std::vector<Core::LinAlg::Matrix<nsd, 1>> flux(0.0);
+        std::vector<Core::LinAlg::Matrix<nsd, 1>> flux(true);
```

Comment:
Here, it is the constructor of the std::vector:

```suggestion
        std::vector<Core::LinAlg::Matrix<nsd, 1>> flux(0);
```